### PR TITLE
Draw attention to transport layer in remote cluster documentation

### PR DESCRIPTION
**Pull request by akki**
https://github.com/elastic/elasticsearch/issues/43858

### CODE REVIEW

**Comment by akki** [comment]
@elasticmachine test this please

File: docs/reference/modules/remote-clusters.asciidoc [hunk 1]
```diff
@@ -63,7 +63,9 @@ to satisfy the two above requirements.
 You can configure remote clusters globally by using
 <<cluster-update-settings,cluster settings>>, which you can update dynamically.
 Alternatively, you can configure them locally on individual nodes by using the
- `elasticsearch.yml` file.
+ `elasticsearch.yml` file. Note that the connection between the clusters uses
+the transport connection string (not the http connection) which is configured
+via the <<modules-transport,transport layer>>.
```
Comment (jimczi) [inline docs/reference/modules/remote-clusters.asciidoc]: This is redundant with the sentence at the beginning so I am not sure this is needed ?

File: docs/reference/modules/remote-clusters.asciidoc [hunk 2]
```diff
@@ -75,20 +77,22 @@ nodes. For example:
...
+            skip_unavailable: true <5>
 
 --------------------------------
 <1> `cluster_one` and `cluster_two` are arbitrary _cluster aliases_ representing
 the connection to each cluster. These names are subsequently used to distinguish
 between local and remote indices.
-<2> A keep-alive ping is configured for `cluster_one`.
-<3> Compression is explicitly enabled for requests to `cluster_two`.
-<4> Disconnected remote clusters are optional for `cluster_two`.
+<2> The connection string to reach the cluster via the
```
Comment (jimczi) [inline docs/reference/modules/remote-clusters.asciidoc]: I would not call this the connection string, maybe something like `The hostname and transport port of a seed node in the remote cluster` ?

**Review by jimczi — CHANGES_REQUESTED** [review]
Thanks @akki , I left some comments

File: docs/reference/modules/remote-clusters.asciidoc (hunk 1 above)
Comment (akki) [reply docs/reference/modules/remote-clusters.asciidoc]: I think if I explicitly mention "host"/"port" as you suggested in your other comment, that would suffice. Thanks! :)

File: docs/reference/modules/remote-clusters.asciidoc [hunk 3]
```diff
@@ -75,20 +75,22 @@ nodes. For example:
...
+            skip_unavailable: true <5>
 
 --------------------------------
 <1> `cluster_one` and `cluster_two` are arbitrary _cluster aliases_ representing
 the connection to each cluster. These names are subsequently used to distinguish
 between local and remote indices.
-<2> A keep-alive ping is configured for `cluster_one`.
-<3> Compression is explicitly enabled for requests to `cluster_two`.
-<4> Disconnected remote clusters are optional for `cluster_two`.
+<2> The hostname and <<modules-transport,transport>> port (default: 9300) of a
```
Comment (akki) [inline docs/reference/modules/remote-clusters.asciidoc]: I think the `default: 9300` part would the trick. :)

**Comment by akki** [comment]
@jimczi Apologies for the delay. I have addressed your comments, please have a look now.

**Review by jimczi — APPROVED** [review]
LGTM, thanks @akki

**Comment by jimczi** [comment]
@elasticmachine ok to test

**Pull request merged**